apl-vault-wallet/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.apollocurrency</groupId>
+        <artifactId>apollo-blockchain</artifactId>
+        <version>1.48.0</version>
+    </parent>
+
+    <artifactId>apl-vault-wallet</artifactId>
+    <name>apl-vault-wallet</name>
```

Comment:
Could you explain please, why we need separate module for vault?

Reply:
It's a first step for KMS implementation.